[ERROR] KeyError: 'exitCode'

```
[ERROR] KeyError: 'exitCode'
Traceback (most recent call last):
  File "/var/task/slack_notifications.py", line 225, in lambda_handler
    slack_message = event_to_slack_message(event)
  File "/var/task/slack_notifications.py", line 139, in event_to_slack_message
    known_detail = ecs_events_parser(detail_type, detail)
  File "/var/task/slack_notifications.py", line 117, in ecs_events_parser
    result = result + '\n' + ' - ' + container['name'] + ': ' + str(container['exitCode'])
```
```
        "containers": [
            {
                "containerArn": "XXXXXX",
                "lastStatus": "STOPPED",
                "name": "XXXXX",
                "image": "XXXXX",
                "runtimeId": "ea6fc90239c745458217609d1290b790-1099699051",
                "taskArn": "XXXXXXX",
                "networkInterfaces": [
                    {
                        "attachmentId": "a10112a0-48ee-497c-b32b-2eaebcd2effc",
                        "privateIpv4Address": "XXXXXX"
                    }
                ],
                "cpu": "0",
                "managedAgents": [
                    {
                        "name": "ExecuteCommandAgent",
                        "status": "STOPPED",
                        "reason": "CannotConfigureManagedAgentError: failed to configure managed agent inside container: timeout while waiting for configuring execute command agent to finish"
                    }
                ]
            },
```

--- a/functions/slack_notifications.py
+++ b/functions/slack_notifications.py
@@ -114,7 +114,7 @@ def ecs_events_parser(detail_type, detail):
             if 'containers' in detail:
                 result = result + '\n' + 'Task containers and their exit code:'
                 for container in detail['containers']:
-                    result = result + '\n' + ' - ' + container['name'] + ': ' + str(container['exitCode'])
+                    result = result + '\n' + ' - ' + container['name'] + ': ' + str(container.get('exitCode', 'unknown'))
         return result
 
     return f'*Event Detail:* ```{json.dumps(detail, indent=4)}```'

